ui/lib/config-ui/addon/components/messages/page/create-and-edit-message-form.hbs

```diff
@@ -9,21 +9,22 @@
 />
 
 <form id="message-create-edit-form" {{on "submit" (perform this.save)}} data-test-form="create-and-edit">
-  <div class="box is-sideless is-fullwidth is-marginless has-top-padding-s">
-    <Hds::Text::Body @tag="p" class="has-bottom-margin-l" data-test-form-subtext>
-      {{if @message.isNew "Create" "Edit"}}
-      a custom message for all users when they access a Vault system via the UI.
-    </Hds::Text::Body>
-
+  <div class="box is-sideless is-fullwidth is-marginless">
+    <NamespaceReminder @mode="create" @noun="message" />
```

Comment:
This message should either only show when `@model.isNew` or the mode should change to `edit` if the model is not new if you want to keep the message for editing